README.md

```diff
@@ -0,0 +1,31 @@
+# FFTPACK
+A package of fortran subprograms for the fast fourier transform of periodic and other symmetric sequences.
+
+## Getting started
+### Get the code
+```bash
+git clone https://github.com/certik/fftpack.git
+cd fftpack
+```
+
+### Build with [fortran-lang/fpm](https://github.com/fortran-lang/fpm)
+Fortran Package Manager (fpm) is a great package manager and build system for Fortran.   
+You can build using provided `fpm.toml`:
+```bash
+fpm build --flag "-O2"
+fpm test --flag "-O2" tstfft
```

Comment:
Because the `fpm` software uses the `-fcheck=bounds` flag option by default, this is not compatible with the source code of fftpack now.
For example in your repo here:
https://github.com/certik/fftpack/blob/030cbf4fd35c35ef19387b4835bde3ac73019b64/cosqb1.f#L3

They shall be:
```fortran
DIMENSION       X(*)       ,W(*)       ,XH(*) 
```
I will fix all of them last time, see my done repo [fortran-fans/fftpack4.0-public-domain](https://github.com/fortran-fans/fftpack4.0-public-domain):

https://github.com/fortran-fans/fftpack4.0-public-domain/blob/bc35769b8eeee86a1dda850cb4fef8cadde62e5f/src/cosqb1.f#L3